pennylane/drawer/tape_mpl.py

```diff
@@ -328,7 +330,7 @@ def tape_mpl(
         measurements = [qml.expval(qml.Z(0))]
         tape = qml.tape.QuantumTape(ops, measurements)
 
-        fig, ax = tape_mpl(tape)
+        fig, ax = qml.drawer.tape_mpl(tape)
```

Comment:
Added the module reference to the function throughout the docstring to make it easier accessible when copying code from docstring.